cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,84 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if billingAccount == "" && !interactive.Enabled() {
+		billingAccount = "765374464689"
+		r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+		r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+	}
+
+	if interactive.Enabled() {
+		cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
+		}
+
+		if len(cloudAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
+				Default:  billingAccount,
+				Required: true,
+				Options:  cloudAccounts,
+			})
+
+			pattern := regexp.MustCompile(`[-]?\d[\d,]*[\.]?[\d{2}]*`)
+			billingAccount = pattern.FindString(billingAccount)
```

Comment:
Can we abstract this to a small function with some unit tests. So its easier to maintain and we understand exactly what that regexp is expecting?

Reply:
Yeah sure, will add some unit texts regarding that. Thanks for the review @ciaranRoche :)